docs/api/process.md

```diff
@@ -7,3 +7,7 @@ upstream node:
 * `process.versions['electron']` String - Version of Electron.
 * `process.versions['chrome']` String - Version of Chromium.
 * `process.resourcesPath` String - Path to JavaScript source code.
+
+## process.hang
+
+Call in the `renderer` process to trigger `unresponsive` event.
```

Comment:
It sill needs to be clear in document, otherwise people would just call it in the main process and hope the renderer process to hang. Just say something like "Makes the main thread of current process hang" is enough.